src/test/java/Problem3Test.java

```diff
@@ -134,4 +134,32 @@ public void testStoreMediaCalcLateFees() {
         assertEquals(expect, fees);
     }
 
+    @Test
+    public void testBugFixEqualsBookRomance(){
+        BookRomance justID = new BookRomance (null,null);
+        Book justID2 = new BookRomance(justID);
+        assertEquals(justID, justID2);
```

Comment:
this doesn't look right. what we are looking for is same id but different author or title.